buildprocess/ci-deploy.sh

```diff
@@ -1,5 +1,10 @@
 #!/bin/sh
 
+# Don't run for greenkeeper branches; there are too many!
+if [[ $TRAVIS_BRANCH =~ ^greenkeeper/ ]]; then
```

Comment:
Does this mean that greenkeeper won't tell us when packages break our build?

Reply:
My bad. Didn't read the full description :)

Reply:
It would be nice to auto-deploy the greenkeeper stuff too, but there are so many the k8s cluster was running out of memory and exploding. A bigger cluster would solve the problem, but I don't think it's worth the cost.